plugins/modules/tmp_vcenter_vm_info.py

```diff
@@ -0,0 +1,43 @@
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "community",
+}
```

Comment:
This is already covered under - `vmware_core_info` module

Reply:
I know, but I would prefer to generate all the modules, this to have a consistent set.